ggml.c

```diff
@@ -89,7 +89,9 @@ static int pthread_create(pthread_t * out, void * unused, thread_ret_t(*func)(vo
 
 static int pthread_join(pthread_t thread, void * unused) {
     (void) unused;
-    return (int) WaitForSingleObject(thread, INFINITE);
+    int csq =(int) WaitForSingleObject(thread, INFINITE);
```

Comment:
There should be a space after the '=' here. And 'csq' doesn't mean anything to me in this context, perhaps 'res' (result) or 'ret' (return value)?

Reply:
ok, I have corrected it